dali/operators/math/expressions/expression_impl_gpu.cuh

```diff
@@ -174,6 +210,33 @@ class ExprImplGPUInvoke : public ExprImplBase {
     auto grid = GetGridLayout(kBlocksX, tiles.size());
     auto block = dim3(kThreadNum, 1, 1);
     Invoker::Invoke(tiles_.data<ExtendedTileDesc>(), grid, block, ctx.stream);
+
+    CUDAEvent start = CUDAEvent::CreateWithFlags(0);//todo(klecki): remove benchmarking
+    CUDAEvent end = CUDAEvent::CreateWithFlags(0);
+
+    Invoker::Invoke(tiles_.data<ExtendedTileDesc>(), grid, block, ctx.stream);
+
+    cudaEventRecord(start, ctx.stream);
+    constexpr int kIters = 100;
+    for (int i = 0; i < kIters; i++) {
+      Invoker::Invoke(tiles_.data<ExtendedTileDesc>(), grid, block, ctx.stream);
+    }
+    cudaEventRecord(end, ctx.stream);
+    cudaDeviceSynchronize();
+    float time;
+    CUDA_CALL(cudaEventElapsedTime(&time, start, end));
+    std::cerr << "Elapsed Time: " << time  << " s\n";
+
+    // time *= (1e+6f / kIters);  // convert to nanoseconds / 100 samples
```

Comment:
?

Reply:
Will remove all of the profiling before posting final version.